src/e2e/java/teammates/e2e/cases/lnp/BaseLNPTestCase.java

```diff
@@ -205,14 +267,18 @@ protected void runJmeter(boolean shouldCreateJmxFile) throws IOException {
         resultCollector.setFilename(resultsFile);
         testPlan.add(testPlan.getArray()[0], resultCollector);
 
-        // Run JMeter Test
         jmeter.configure(testPlan);
         jmeter.run();
 
-        // TODO: As mentioned in the docs, good to fail the test if there is an Exception, or if the `success` value of
-        //  requests is `false`. An example of when this occurs is if the JMeter test properties are not set or if `email`
-        //  is used for logging in instead of `googleid`. Tests should fail if this assertion fails:
-        //      assertTrue(resultsErrorRate < this.getAcceptableErrorRate());
+        try {
+            ReportGenerator reportGenerator = new ReportGenerator(resultsFile, null);
+            reportGenerator.generate();
+        } catch (Exception e) {
```

Comment:
I think we should catch the exception similar to how we catch exception if a file cannot be renamed and if IOException occurs in `createTestData()`?

In the generate function for `ReportGenerator`, it will throw an error if there is error in generating the report.

```
try {
        source.run(); // NOSONAR
    } catch (SampleException ex) {
        throw new GenerationException("Error while processing samples:"+ex.getMessage(), ex);
    }
```

Reply:
Alright, sounds good. But, let's make it more specific and change `Exception` to `GenerationException`?